Makefile

```diff
@@ -1,6 +1,10 @@
 build:
```

Comment:
Consider adding dependencies on the Dockerfile and any contents that are brought in via `ADD` or `COPY`. This will help Make to determine when it needs to re-run the command.

One approach to ensure this would be to have an arrangement like:

```make
.docker-build: Dockerfile ...
    docker build -t ffig/web-base:latest .
    touch .docker-build

build: .docker-build

run: build
```